scipy/sparse/linalg/_eigen/tests/test_svds.py

```diff
@@ -144,18 +144,16 @@ class SVDSCommonTests:
     def test_svds_input_validation_A(self, args):
         A, error_type, message = args
         with pytest.raises(error_type, match=message):
-            svds(A, k=1, solver=self.solver)
+            svds(A, k=1, solver=self.solver, rng=0)
 
     @pytest.mark.parametrize("which", ["LM", "SM"])
     def test_svds_int_A(self, which):
         A = np.asarray([[1, 2], [3, 4]])
         if self.solver == 'lobpcg':
             with pytest.warns(UserWarning, match="The problem size"):
-                res = svds(A, k=1, which=which, solver=self.solver,
-                           random_state=0)
+                res = svds(A, k=1, which=which, solver=self.solver, rng=0)
         else:
-            res = svds(A, k=1, which=which, solver=self.solver,
-                       random_state=0)
+            res = svds(A, k=1, which=which, solver=self.solver, random_state=0)
```

Comment:
```suggestion
            # preserve a use of legacy keyword `random_state` during SPEC 7 transition
            res = svds(A, k=1, which=which, solver=self.solver, random_state=0)
```

Reply:
Wherever it is, it has been suggested that we leave a comment so it's clear that it's intentional. It appears to be here, do I suggested adding the comment.

Reply:
I put the same comment that you suggested in the other location (line 178).